README.md

```diff
@@ -42,10 +42,12 @@ To run the app from source, just clone the code and run this in your terminal:
 
 ```
 npm install
-npm run build
```

Comment:
Probably shouldn't remove this, especially as we aren't keeping the build in git anymore. If you download the repository and just run npm install and npm start, you won't get the built files, will you?